integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -422,6 +422,43 @@ public static ExecResult kubectlexecNoCheck(String podName, String namespace, St
     return ExecCommand.exec(cmdKubectlSh.toString());
   }
 
+  public static void copyAppFilesToPod(
```

Comment:
javadoc missing

Reply:
added
   * @param appLocationOnHost - App location on the local host
   * @param appLocationInPod - App location on the lk8s pod
   * @param podName - the k8s pod name
   * @param namespace - namespace the k8s pod is in